sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -591,6 +592,8 @@ def __init__(
         batch has not been completely filled up.
       retry_strategy: The strategy to use when retrying streaming inserts
         into BigQuery. Options are shown in bigquery_tools.RetryStrategy attrs.
+      additional_bq_parameters (dict, callable): A set of additional parameters
+        to be passed when creating a BigQuery table.
```

Comment:
Also, prob. explain what users are expected to pass here and how they will be used. It's not clear to an end user how this parameter should be constructed and how this will be used by reading the current doc string.